server/config/config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "root",
+    "password": null,
+    "database": "database_development",
```

Comment:
`test` and `production` are fine for now since we're not using them, but you should probably change these values.

I'd recommend using `process.env.TUBE_TRACKER_PG_USERNAME || "root"` and `process.env.TUBE_TRACKER_PG_PASSWORD || null` or something. Not all of us will have the same configuration.

As for the `database`, `database_development` is pretty vague. I would change it to `tube_tracker_development` so it's more descriptive.